sdk/python/foundation-models/system/inference/text-generation/llama-safe-online-deployment.ipynb

```diff
@@ -502,21 +439,20 @@
     "    \"UAI_CLIENT_ID\": uai_client_id,\n",
     "}\n",
     "\n",
-    "if hf_tgi:\n",
-    "    # Environment variables that are used by the HuggingFace text-generation-inference launcher\n",
-    "    model_environment_vars = {\n",
-    "        \"MAX_INPUT_LENGTH\": \"2048\",\n",
-    "        \"MAX_TOTAL_TOKENS\": \"4096\",\n",
-    "        \"NUM_SHARD\": 1,  # Sharding requires Flash Attention which doesn't seem to work on non-A100 GPUs\n",
-    "        # \"SHARDED\": \"true\",  # Comment `NUM_SHARD` above and uncomment this, if you are running on A100\n",
-    "        # \"QUANTIZE\": \"\",\n",
-    "        # \"DTYPE\": \"float16\",\n",
-    "        # \"TRUST_REMOTE_CODE\": \"true\",\n",
-    "        # \"MAX_CONCURRENT_REQUESTS\": \"\",\n",
-    "        # \"MAX_BEST_OF\": \"\",\n",
-    "        # \"MAX_STOP_SEQUENCES\": \"\",\n",
-    "    }\n",
-    "    deployment_env_vars.update(model_environment_vars)"
+    "# Following environment variables are available for use by the HuggingFace text-generation-inference launcher\n",
+    "model_environment_vars = {\n",
+    "    \"NUM_SHARD\": 2,  # Sharding requires Flash Attention which doesn't work on non-A100 GPUs\n",
```

Comment:
Why do we still need to set these?

Reply:
Let me remove this, i forgot. Thanks for the callout. I think I kept the commented lines just for a sample (i.e. what all can be passed through env variables).